### DISCUSSION TITLE
Line Spacing

### DISCUSSION THREAD

**Discussion by 2023PHOENIX** [cpeditor/cpeditor#1122] [Ideas]
Why there is no line spacing option in editor
![image](https://user-images.githubusercontent.com/55507908/204118228-d2fec4eb-69de-42ba-9e83-69aa78bb7942.png)

this is looking more cleaner
![image](https://user-images.githubusercontent.com/55507908/204118239-0cdd81e8-29ad-4484-8e5f-10efdf9ea92a.png)

**Comment by ouuan** [comment]
It has been requested in #1062. However, it's difficult to elegantly change line spacing in Qt.
